code/modules/power/supermatter/supermatter.dm

```diff
@@ -304,6 +306,12 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			visible_message("<span class='warning'>[src] melts through [T]!</span>")
 		return
 
+	if(last_complete_process > SSair.last_complete_process) 
+		power_changes = FALSE //Atmos has not been fully processed since the previous time the SM was. Abort all power and processing operations.
```

Comment:
erm... wasn't power_changes removed?

Reply:
No?

Previously the variable was a static boolean - true for hugbox SM, and false for all others. I stole the variable name due to its redundance with `processes` instead of just making a new one.